onnxscript/ir/_core.py

```diff
@@ -670,6 +670,12 @@ def tobytes(self) -> bytes:
         length = self._length or self.nbytes
         return self.raw[offset : offset + length]
 
+    def cleanup(self) -> None:
+        """Delete all references to the memory buffer and close the memory-mapped file."""
+        self._array = None
+        assert self.raw is not None
+        self.raw.close()
```

Comment:
```suggestion
    def release(self) -> None:
        """Delete all references to the memory buffer and close the memory-mapped file."""
        self._array = None
        if self.raw is not None:
            self.raw.close()
            self.raw = None
```

Reply:
Could you add a test to verify that calling numpy() etc. on a released tensor will still work (e.g. we can release and load again at will)? Thanks!